macro/src/lib.rs

```diff
@@ -153,6 +167,19 @@ pub fn subxt(args: TokenStream, input: TokenStream) -> TokenStream {
         )
     }
 
+    let mut type_substitutes = TypeSubstitutes::new(&crate_path);
+    type_substitutes.extend(args.substitute_type.into_iter().map(
+        |SubstituteType { ty, with }| {
+            (
+                ty,
+                with.try_into()
```

Comment:
This can be quite cryptic, but I didn't want to import the `AbsoluteTypePath` newtype wrapper that I introduced for the validation purpose (rather than it being a more well-rounded/useful type on its own), to name it directly here.